fapolicy_analyzer/glade/main_window.glade

```diff
@@ -249,6 +282,63 @@
                 <property name="position">2</property>
               </packing>
             </child>
+            <child>
+              <object class="GtkStatusbar">
+                <property name="visible">True</property>
+                <property name="can-focus">False</property>
+                <property name="margin-left">10</property>
+                <property name="margin-right">10</property>
+                <property name="margin-start">10</property>
+                <property name="margin-end">10</property>
+                <property name="margin-top">6</property>
+                <property name="margin-bottom">6</property>
+                <property name="spacing">2</property>
+                <child>
+                  <object class="GtkProgressBar" id="fapaStatusProgressBar">
+                    <property name="visible">False</property>
+                    <property name="can-focus">False</property>
+                    <property name="fraction">0.0</property>
+                    <property name="text" translatable="yes">Percent Complete</property>
+                    <property name="show-text">False</property>
+                  </object>
+                  <packing>
+                    <property name="expand">True</property>
+                    <property name="fill">True</property>
+                    <property name="position">0</property>
+                  </packing>
+                </child>
+                <child>
+                  <object class="GtkLabel">
```

Comment:
Would be cool if instead of an icon for the status it was a `GtkSwitch` control, that would both indicate status and allow control from the main GUI.

Reply:
I'm Ok w/that but next iteration of the status bar, would be my preference.

Reply:
Opened  "Convert fapolicyd status bar image icon to GtkSwitch widget" #370